lang/lang_test.go

```diff
@@ -11,14 +11,14 @@ import (
 func TestAddTranslations(t *testing.T) {
 	setupLang("en")
 	err := AddTranslations(fyne.NewStaticResource("en.json", []byte(`{
-  "Test": "Match"
-}`)))
+		"Test": "Match"
```

Comment:
This doesn't seem right - why would a single indentation of a JSON string be 2 tabs?

Reply:
JSON does not bother about indentation at all. This is not something written into a text file and formatted to be readable there. It is something that should be readable right here in the test code.

Reply:
Ah, thanks - I thought it was a comparison sorry